compiler/src/iree/compiler/Codegen/Common/test/math_transform.mlir

```diff
@@ -1,6 +1,9 @@
 // RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-codegen-math-transform))' --split-input-file %s | FileCheck %s
+// RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-codegen-math-transform))' '--iree-codegen-math-transform-tweaks=-approx:math.tan,-rewrite:math.tan,-f32cast:math.tan' --split-input-file %s | FileCheck %s --check-prefix=FLAGS1
+// RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-codegen-math-transform))' '--iree-codegen-math-transform-tweaks=-approx:math.tan,-rewrite:math.tan,+f32cast:math.tan' --split-input-file %s | FileCheck %s --check-prefix=FLAGS2
+// RUN: iree-opt --pass-pipeline='builtin.module(func.func(iree-codegen-math-transform))' '--iree-codegen-math-transform-tweaks=-approx:math.tan,-approx:math.cos,-approx:math.sin,+rewrite:math.tan' --split-input-file %s | FileCheck %s --check-prefix=FLAGS3
 
-// CHECK-LABEL: @rewrite_tan
+// CHECK-ALL-LABEL: @rewrite_tan
```

Comment:
Do I misread this? I think it does not check anything because you didn't put `CHECK-ALL` to the `--check-prefix`.

You also need to update the `--check-prefix` for the first line. E.g.,

https://github.com/iree-org/iree/blob/50844fa6d447894608861f75d44957f1f6ec84ed/compiler/src/iree/compiler/Codegen/Common/test/decompose_pack_unpack_ops.mlir#L1-L2

Reply:
Thanks for the catch!